src/frontend/components/UI/QuestsViewer/components/QuestDetailsWrapper/index.tsx

```diff
@@ -25,23 +31,82 @@ import { getRewardCategory } from 'frontend/helpers/getRewardCategory'
 import { getDecimalNumberFromAmount } from '@hyperplay/utils'
 import { useFlags } from 'launchdarkly-react-client-sdk'
 import { getPlaystreakArgsFromQuestData } from 'frontend/helpers/getPlaystreakArgsFromQuestData'
+import { createPublicClient } from 'viem'
+import { chainMap, parseChainMetadataToViemChain } from '@hyperplay/chains'
+import { InfoAlertProps } from '@hyperplay/ui/dist/components/AlertCard'
 
 export interface QuestDetailsWrapperProps {
   selectedQuestId: number | null
 }
 
+const averageGasPerFunction: Record<string, number> = {
```

Comment:
it isn't the average gas per function though right? it's the estimated gas per function here. maybe we should rename

Reply:
that is the average gas cost to execute the function, we can add estimated to the name

Reply:
![image](https://github.com/user-attachments/assets/5a4f633b-b71f-49d8-9daf-fdaf3f87e1c4)

do you say we use the max?